datadog/resource_datadog_dashboard.go

```diff
@@ -995,9 +995,106 @@ func getNonGroupWidgetSchema() map[string]*schema.Schema {
 				Schema: getRunWorkflowDefinitionSchema(),
 			},
 		},
+		"split_graph_definition": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			MaxItems:    1,
+			Description: "The definition for a Split Graph widget.",
+			Elem: &schema.Resource{
+				Schema: getSplitGraphDefinitionSchema(),
+			},
+		},
 	}
 }
 
+func getSplitGraphSourceWidgetSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		// A widget should implement exactly one of the following definitions
```

Comment:
Interesting! 
I thought that it would be invalidated, because we set minItems & maxItems in here
```
		"source_widget_definition": {
			Description: "The original widget we are splitting on.",
			Type:        schema.TypeList,
			Required:    true,
			MinItems:    1,
			MaxItems:    1,
			Elem: &schema.Resource{
				Schema: getSplitGraphSourceWidgetSchema(),
			},
		```

Reply:
Fixed in here https://github.com/DataDog/terraform-provider-datadog/pull/2129/commits/7e800b95ec706121b279b07c149860e2a73fc0b0

Reply:
Nice! Double checked it worked as expected:
```
 Error: failed to parse resource configuration: source widget definition must contain exactly one value
```